02-your-first-test/src/test/java/tests/TestLogin.java

```diff
@@ -13,6 +13,12 @@
 
     @Before
     public void setUp() {
+
+        // Windows
+        //System.setProperty("webdriver.gecko.driver", "C:\\tools\\selenium\\geckodriver.exe");
+
+        // Mac
+        //System.setProperty("webdriver.gecko.driver", "/usr/local/bin/geckodriver");
```

Comment:
This is good info to have commented out like this (+1), but it's contradictory to what you do in the assertions section (by using Chrome instead). I would just switch the order of the browsers completely, starting everything off using Chrome. Later, when talking about cross-browser testing, demonstrate adding in geckodriver.